### DISCUSSION TITLE
How to use Decimal column type with migration

### DISCUSSION THREAD

**Discussion by jackieit** [SeaQL/sea-orm#871] [Q&A] [answered]
my column type in entity is 
 
pub struct Model {
    #[sea_orm(primary_key)]
    pub id: i32,
    pub price: Option<Decimal>,
    ... and more
}

 

I create table with migration ,but when compled column the field  type in table  is decimal(10,0) 
if I want column type in table is decimal(10,6) , how to set  .
Thanks.

**Comment by jackieit** [comment, marked as answer]
Find the answer ,in migration file use decimal_len(10,6) method.

**Comment by billy1624** [comment]
Thanks for the updates! @jackieit
